azure-quantum/azure/quantum/target/__init__.py

```diff
@@ -1,4 +1,30 @@
-from azure.quantum.target.ionq import IonQ
-from azure.quantum.target.honeywell import Honeywell
-from azure.quantum.target.fleet_management import MicrosoftFleetManagement
-from azure.quantum.target.target import Target
+from .target import Target
```

Comment:
we should add default targets for other 3pp: toshiba an 1qbit.

Reply:
Thanks for catching this!